### PR TITLE
Add backend

### PR DESCRIPTION
**Pull request by deloppa**
### What does it change?

Add initial project structure (documentation and .gitignore files).
Implement basic functionality for the FastAPI app platform.

(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)😢🐱‍💻O_O:O)~_~O.O=P:'(:S¬_¬^_+🤶🧀🍣🎀🎄🎄🎄🎄🎄✨✨✨✨✨✨✨🏀🏀(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)(-_-) (0_0)😢🐱‍💻O_O:O)~_~O.O=P:'(:S¬_¬^_+🤶🧀🍣🎀🎄🎄🎄🎄🎄✨✨✨✨✨✨✨🏀🏀🏀🏀🏀🏀🏀🤿🤿🤿🤿🤿🤿🏀🏀🏀🏀🏀🤿🤿🤿🤿🤿🤿

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Do not use the same values in the non-dev environments!
+
+MARIADB_ROOT_PASSWORD="8HrA2pzlmmgU3Avi"
+MARIADB_DATABASE="svitbo"
+MARIADB_USER="admin"
+MARIADB_PASSWORD="Q9bl907JI4y8"
+
+INITIAL_ADMIN_USERNAME="kirby"
```
Comment (ghost) [inline .env.example:8]: Why is it `kirby`? Maybe let's choose something more serious like `admin`?

Comment (deloppa) [reply .env.example:8]: It's a security feature.

Comment (ghost) [reply .env.example:8]: Well, idk about that, but let it stay as it is.

**Review by ghost — APPROVED** [review]
Looks good to me

**Pull request merged**